### PR TITLE
Pranita_Assignment_24_Done_1Hour

### PR DESCRIPTION
**Pull request by PranitaPJun20**
image

### CODE REVIEW

File: src/pranitaPJun20/ValidPhoneNumber.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+						|| phoneNumber.startsWith("9"))) {
+					System.out.println("\nPhone number " + phoneNumber + " having length "+length +" is a valid Phone Number");
+				} else {
+					System.out.println("\n" + phoneNumber + " is not a valid Phone Number.");
+				}
+				break;
+
+			case 11:
+					if (phoneNumber.startsWith("0") && checkIsDigit(phoneNumber) == true && (phoneNumber.charAt(1) =='7' || phoneNumber.charAt(1) =='8'
+							|| phoneNumber.charAt(1) =='9')) {
```
Comment (harshvegada) [inline src/pranitaPJun20/ValidPhoneNumber.java]: Here you should use startsWith instaed of chatAt bcoz charAt() index might be change

**Review by harshvegada — COMMENTED** [review]
please work on comments rest logic is as per the expetations

File: src/pranitaPJun20/ValidPhoneNumber.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/ValidPhoneNumber.java]: Changes done as suggested.

**Review by harshvegada — APPROVED** [review]
Looks Good Code @PranitaPJun20

**Pull request merged**